broker/submissions/routes.py

```diff
@@ -0,0 +1,35 @@
+import tempfile
+import time
+
+import jsonpickle
+from flask import Blueprint, send_file
+from flask import current_app as app
+
+from broker.service.summary_service import SummaryService
+from broker.submissions.export_to_spreadsheet_service import ExportToSpreadsheetService
+
+submissions_bp = Blueprint(
+    'submissions', __name__, url_prefix='/submissions'
+)
+
+
+@submissions_bp.route('/<submission_uuid>/spreadsheet', methods=['GET'])
+def export_to_spreadsheet(submission_uuid):
+    workbook = ExportToSpreadsheetService(app.ingest_api).export(submission_uuid)
+    timestamp = time.strftime("%Y%m%d-%H%M%S")
+    temp_file = tempfile.NamedTemporaryFile()
+    filename = f'{submission_uuid}_{timestamp}.xlsx'
```

Comment:
What happens fi 2 users try to export the same project? can this cause a problem? Do we need to make this in a temp directory?

Reply:
it shouldn't be a problem, there is a timestamp (up to seconds) in the filename as well. If that happens the user can just try downloading again. It's already a tempfile object. They get cleaned up automatically.